src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -166,9 +166,7 @@ def storage_custom_command_oauth(self, *args, **kwargs):
     def get_handler_suppress_some_400(self):
         def handler(ex):
```

Comment:
Looks like it's not a handler anymore?

Reply:
It can't handle exceptions. As the design of Knack, no other handler should mute the exception or Knack will have error. See the description of the PR.